docs/educational-resources/beginner/Articles/how-to-stake.mdx

```diff
@@ -1,22 +1,78 @@
+
+To demonstrate the staking process, we will use a compatible wallet with sufficient funds. For security reasons, you should store your tokens in cold storage wallets and reserve hot storage wallets exclusively for staking purposes.
+ 
+## How to Swap Crypto Native Tokens to 1INCH?
+
+Prior to staking on the 1inch Protocol, it is essential to exchange your crypto native tokens for 1INCH tokens. By doing so, you will obtain the necessary authorization to stake and earn rewards.
+
+Below are the steps to convert your native tokens to 1INCH tokens:
+
+1. Go to [1inch.io](https://1inch.io/).
+2. Locate the **Launch DApp** button situated in the upper right corner and click on it. This action will immediately take you to the swap mode interface.
+3. Click on the **Connect Wallet** button positioned in the top right corner to connect your wallet.
+4. Choose your preferred native token by selecting a chain from the chains' dropdown menu beside the **Connect Wallet** button.
+5. Indicate the amount of your desired native tokens you wish to convert to 1INCH tokens.
+6. Proceed by clicking on the **Swap** button.
+7. Confirm the swap by clicking on the **Confirm Swap** button.
+8. Finally, validate the transaction within your wallet to complete the process.
+
+## Staking 1INCH Tokens
+
+Once you have acquired a certain amount of 1INCH tokens, you can to stake them by following these steps:
+
+1. Go to the DAO drop-down menu located in the navigation bar.
+2. Click on the **Staking** option from the menu, which will redirect you to the staking interface.
 
 <DropDown/>
 
-Next, enter the amount of 1INCH you would like to stake, set the locking period (default is 2 years) and click “Give permission to stake”. Then sign the transaction in your connected wallet.
+
+3. Enter your desired amount of tokens you intend to stake.
+4. Click on the **Give Permission** button and validate the transaction within your wallet.
 
 ![](./assets/GivePermissionButton.jpeg)
 
+5. Allow some time for the transaction to be processed and confirmed on the blockchain.
+6. Click on the **Stake** button and confirm the transaction within your wallet. Subsequently, you will be able to view the total amount of staked tokens, the lock period, and your total Unicorn Power, which can be utilized for delegation or 1inch DAO governance voting.
+
+![](./assets/StakingAndUnicornPower.png)
+
+## Troubleshooting and Tips
+
+If you encounter any issues during the staking process, consider the following tips:
+
+- Ensure that your wallet is securely connected and contains an adequate balance.
+- Double-check all transaction details before confirming to prevent any errors.
+- If you encounter any technical challenges, seek assistance from the 1inch Protocol's support resources or community for prompt resolution.
+
+## Unstaking Penalties
```

Comment:
"Penalties" should be moved above "How to stake" section, as users must understand this information before committing to a lock period.